docs/setup-prerequisites.md

```diff
@@ -103,6 +103,14 @@ After that, certificate should be able to be created by using the command (the `
 certbot certonly --non-interactive -d 'tf2pickup.fi' -d '*.tf2pickup.fi' --dns-cloudflare --dns-cloudflare-credentials /root/.secrets/cloudflare --rsa-key-size 4096 --must-staple --agree-tos --email your-mailbox@you-are-really-using.com
 ```
 
+There is a special case if your domain is a subdomain of the `tf2pickup.org` or `tf2pickup.eu`, for instance `br.tf2pickup.org` or `au.tf2pickup.org` which you have to use for (the example given is for `br.tf2pickup.org`).
```

Comment:
Grammar: drop the

> which you have to use for

Reply:
Also, drop the last "the" in `...if your domain is a subdomain of the`